lib/cloudregister/registerutils.py

```diff
@@ -30,6 +30,7 @@
 import subprocess
 import sys
 import time
+import toml
```

Comment:
Starting with Python 3.11, a TOML reader is part of the standard library. That said, a perfectly good module for reading and modifying TOML is tomlkit (which was created by the Poetry developers and used by Poetry itself).